src/Add_remove.js

```diff
@@ -0,0 +1,36 @@
+const TodoLoad = (todos) => {
+  todos = JSON.parse(localStorage.getItem('todos')) || [];
+  for (let i = 0; i < todos.length; i += 1) {
+    todos[i].index = i + 1;
+  }
+  return todos;
+};
+
+class Todo {
+  constructor(description, index = 0, completed = false) {
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
+  }
```

Comment:
I think it is a good practice to separate this class file from other functions because they are totally independent codes. if you split this `class Todo` from other functions it will be easy to troubleshoot your code.